.project-keeper.yml

```diff
@@ -10,7 +10,7 @@ build:
   freeDiskSpace: false
   exasolDbVersions:
     - "8.26.0"
-    - "7.1.27"
+    - "7.1.26"
```

Comment:
Why do you revert to a previous version?
Latest docker db version for 7.1 on dockerhub is 7.1.29.
If there are some limitations of newer version, please add a comment.